x-pack/plugins/rule_registry/server/alert_data_client/alerts_client.ts

```diff
@@ -846,6 +851,51 @@ export class AlertsClient {
     });
   }
 
+  public async removeAlertsFromCase({ caseId, alerts }: RemoveAlertsFromCaseOptions) {
+    try {
+      if (alerts.length === 0) {
+        return;
+      }
+
+      const mgetRes = await this.ensureAllAlertsAuthorized({
+        alerts,
+        operation: ReadOperations.Get,
+      });
+
+      const painlessScript = `if (ctx._source['${ALERT_CASE_IDS}'] != null && ctx._source['${ALERT_CASE_IDS}'].length > 0) {
+        for (int i=0; i < ctx._source['${ALERT_CASE_IDS}'].length; i++) {
+            if (ctx._source['${ALERT_CASE_IDS}'][i] == '${caseId}') {
+                ctx._source['${ALERT_CASE_IDS}'].remove(i);
+            }
+        }
+      }`;
+
+      const bulkUpdateRequest = [];
+
+      for (const doc of mgetRes.docs) {
+        bulkUpdateRequest.push(
+          {
+            update: {
+              _index: doc._index,
+              _id: doc._id,
+            },
+          },
+          {
+            script: { source: painlessScript, lang: 'painless' },
```

Comment:
nice script name 😄

Reply:
Yes because is a pain to write them hahhaha.